File: C++/IsPrime.cpp
#include <iostream>
using namespace std;

int isPrime(int n) {
    int count = 0;

    for(int i = 2; i <= n/2; i++){      //loop from 2 to n/2
        if(n % i == 0){                 //if number gets remainder 0 then its not prime num
            count++;
        }
    }

    if(count == 0){
        return true;
    }
    else{
        return false;
    }
}

int main(){

    int n; 

    cout << "Enter value of n: ";
    cin >> n;

    if(isPrime(n)){                             //Return boolean true
        cout << n << " is Prime Number";
    }
    else{                                       //Return boolean false
        cout << n << " is not Prime Number";
    }

    return 0;
}

File: Java/isPrime.java
import java.util.Scanner;

public class isPrime {

    public static boolean Prime(int n) {

        if (n == 2)
            return true;

        for (int i = 2; i < (n - 1); i++) {
            if (n % i == 0) {
                return false;
            }
        }
        return true;
    }

    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        System.out.println("Enter value of n: ");
        int n = sc.nextInt();

        if (Prime(n)) {
            System.out.println("The number " + n + " is Prime");
        } else {
            System.out.println("The number " + n + " is not Prime");
        }
    }
}
